app/services/message_sender.rb

```diff
@@ -1,9 +1,10 @@
 require "slack-ruby-client"
 
 class MessageSender
-  def initialize(employee, message)
+  def initialize(employee, message, options = {})
```

Comment:
thoughts on using a keyword arg that defaults to `nil` rather than an options hash? Passing the `test_message` arg would look exactly the same. What I like about using keyword args is that you do need to pass them with the correct key, whereas options hash lets you pass anything 

https://robots.thoughtbot.com/ruby-2-keyword-arguments